frontend/src/metabase/search/components/SearchFilterModal/SearchFilterModal/SearchFilterModal.tsx

```diff
@@ -32,11 +29,30 @@ export const SearchFilterModal = ({
 }) => {
   const [output, setOutput] = useState<SearchFilters>(value);
 
+  const filterMap: Record<FilterTypeKeys, SearchFilterComponent> = useMemo(
+    () => ({
+      [SearchFilterKeys.Type]: TypeFilter,
+      [SearchFilterKeys.Verified]: PLUGIN_CONTENT_VERIFICATION.VerifiedFilter,
+    }),
+    [],
+  );
```

Comment:
Looks like it doesn't rely on any prop here, should we move it outside the component as a constant? That way, we don't need to memoize it either.

Reply:
I'll clean this up in a separate PR (https://github.com/metabase/metabase/pull/33120#discussion_r1295610973)